examples/logger.py

```diff
@@ -3,20 +3,60 @@
 """
 
 import time
-from typing import Any, Optional, Tuple
+import logging
+from typing import Any, Optional, Tuple, Callable
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+
+class Signaller(QObject):
```

Comment:
I still wonder why this class is necessary?
It has only one signal and I think you can move it into `LoggingHandler`.

Reply:
Is there any discussion about this with Jiyong?

Reply:
Its because pyQt5.Qtcore.pyqtSignal object has no attribute 'connect'. So we need to set Qobject as parent of LoggingHandler to use signal.connect. So we need to set Logging.Handler and Qobject as parent of LoggingHandler, but @kangz12345 said that multiple inheritance may be misleading. 
He pointed out vague meaning for initialization function.

Reply:
Oh, I understand now!
Then, how about adding `_` at the begining of the class name? (Just rename `_Signaller`)
It is only for `LoggingHandler`, so it can be treated as a private class.